### PR TITLE
Add example connection using web3j

### PR DESCRIPTION
**Pull request by antonydenyer**
## Pull Request Description

Add examples for how to use web3j when connecting to GoQuorum when using multi-tenancy and auth.

## Fixed Issue(s)
fixes: https://github.com/web3j/web3j-quorum/issues/59

### CODE REVIEW

**Comment by antonydenyer** [comment]
Not sure if we want to start putting client library examples (that we don't control) in our docs, but wanted to start the conversation.

File: docs/concepts/multi-tenancy.md [hunk 1]
```diff
@@ -130,3 +130,14 @@ In order of precedence, users have the following three options to specify a priv
 1. HTTP header. Every RPC request must have an HTTP header "PSI" attached that specifies the private state to use.
 
 1. IPC and inproc connections. Prepend the PSI to the ID field of the `jsonrpcMessage`.
+
+### Accessing a private state using web3j
+
+To access private state using web3j you need to specify the query parameter in the URL when initialising the `HttpService`. It's also likely that you will have auth enabled and will need to specify a bearer token in the `Authorization` header.
```
Comment (alexandratran) [inline docs/concepts/multi-tenancy.md:136]:
```suggestion
To access private state using [web3j](https://github.com/web3j/web3j), specify the query parameter in the URL when initializing the `HttpService`. If you have authentication enabled you must specify a bearer token in the `Authorization` header.
```

File: docs/configure-and-manage/manage/json-rpc-api-security.md [hunk 2]
```diff
@@ -55,6 +55,17 @@ let provider = new HttpHeaderProvider('https://...', headers);
 web3.setProvider(provider);
 ```
 
+### `web3j-quorum`
+
+Using web3j-quorum Java library:
```
Comment (alexandratran) [inline docs/configure-and-manage/manage/json-rpc-api-security.md:60]:
```suggestion
Use the [web3j-quorum](https://github.com/web3j/web3j-quorum) Java library:
```

Can you also change the previous section description to
```
Use the [web3.js](https://web3js.readthedocs.io/en/v1.5.2/) JavaScript library:
```

**Review by alexandratran — COMMENTED** [review]
Suggestions.

File: docs/concepts/multi-tenancy.md [hunk 3]
```diff
@@ -130,3 +130,14 @@ In order of precedence, users have the following three options to specify a priv
 1. HTTP header. Every RPC request must have an HTTP header "PSI" attached that specifies the private state to use.
 
 1. IPC and inproc connections. Prepend the PSI to the ID field of the `jsonrpcMessage`.
+
+### Accessing a private state using web3j
+
+To access private state using [web3j](https://github.com/web3j/web3j), specify the query parameter in the URL when initializing the `HttpService`. If you have authentication enabled you must specify a bearer token in the `Authorization` header.
```
Comment (bgravenorst) [inline docs/concepts/multi-tenancy.md:136]:
```suggestion
To access a private state using [web3j](https://github.com/web3j/web3j), specify the query parameter in the URL when initializing the `HttpService`. If you have authentication enabled you must specify a bearer token in the `Authorization` header.
```

File: docs/configure-and-manage/manage/json-rpc-api-security.md [hunk 4]
```diff
@@ -55,6 +55,17 @@ let provider = new HttpHeaderProvider('https://...', headers);
 web3.setProvider(provider);
 ```
 
+### `web3j-quorum`
+
+Use the [web3j-quorum](https://github.com/web3j/web3j-quorum) Java library:
+
+```java
+HttpService service = new HttpService("https://...");
```
Comment (bgravenorst) [inline docs/configure-and-manage/manage/json-rpc-api-security.md:63]:
```suggestion
HttpService service = new HttpService("<JSON-RPC HTTP endpoint>");
```

Comment (bgravenorst) [reply docs/configure-and-manage/manage/json-rpc-api-security.md:63]: We generally place replaceable code between the "<>" characters. Not sure if my description is correct though

**Review by bgravenorst — APPROVED** [review]
Minor suggestions.

File: docs/configure-and-manage/manage/json-rpc-api-security.md (hunk 4 above)
Comment (antonydenyer) [reply docs/configure-and-manage/manage/json-rpc-api-security.md:63]: Want to nudge people to wards using https when using auth

**Pull request merged**